coalib/bears/Bear.py

```diff
@@ -266,7 +266,9 @@ def __json__(cls):
         """
         Override JSON export of ``Bear`` object.
         """
-        _dict = get_public_members(cls)
+        # json cannot serialize properties, so drop them
```

Comment:
shouldn't the JSONEncoder use the value of the property? It makes a getattr on it which retrieves the value, right?